### ISSUE TITLE
failed to send reset password link

### ISSUE THREAD

**Issue by yuchan** [SocioQuiz/socio_quiz#27]
when I pushed "send me reset password instruction" button, the app crashed.

**Comment by yuchan** [comment]
we maybe need email setting in production env.
like [this](http://stackoverflow.com/a/6069859)

**Comment by not001praween001** [comment]
Yeah! We need the email for mailing system of the account managing feature. 
And we can use it to create OAuth provider account too. (y)
Let discussion on the next meeting.
